src/plugins/vis_type_vega/public/vega_type.ts

```diff
@@ -18,19 +18,24 @@
  */
 
 import { i18n } from '@kbn/i18n';
-import { BaseVisTypeOptions } from 'src/plugins/visualizations/public';
+import { parse } from 'hjson';
+import type { BaseVisTypeOptions } from 'src/plugins/visualizations/public';
+
 import { DefaultEditorSize } from '../../vis_default_editor/public';
-import { VegaVisualizationDependencies } from './plugin';
+import type { VegaVisualizationDependencies } from './plugin';
```

Comment:
just sharing my thoughts for the particular case:
there is no necessity to use `import type` here, since typescript knows it's used as a type and does not include the reference into the bundle.  `import type`  is really useful for re-exporting class declaration as types, since the compiler doesn't know it's actual usage